RPM build fails

I am starting from the point where make succeeds (source code from master branch, latest commit f448472)

When I do `make rpm` I get the following error:
```
libtool: link: gcc -ggdb -I../include -I../nfdump/bin -I../src/ffilter -I../bzip2 -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -mtune=generic -DLNF_THREADS -DNSEL -Wl,-z -Wl,relro -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o libnf-info libnf-info.o  ../src/.libs/libnf.a -lresolv -lpthread
/usr/bin/ld: ../src/.libs/libnf.a(bzlib.o): relocation R_X86_64_32S against `BZ2_crc32Table' can not be used when making a shared object; recompile with -fPIC
../src/.libs/libnf.a: error adding symbols: Bad value
collect2: error: ld returned 1 exit status
Makefile:575: recipe for target 'libnf-info' failed
make[4]: *** [libnf-info] Error 1
```

I'll try to add -fPIC and let you know.

--- a/libnf.spec.in
+++ b/libnf.spec.in
@@ -44,7 +44,7 @@ Nfdumpp is implementation of parallel nfdump based on libnf.net library.
 
 
 %build
-%configure
+%configure CFLAGS=-fPIC
 make %{?_smp_mflags}
 
 

